cmds/core-service/README.md

```diff
@@ -40,11 +40,11 @@ docker container run -p 26257:26257 -p 8080:8080 --rm cockroachdb/cockroach:v24.
 Once an initialized CockroachDB cluster is available, the necessary databases within the CRDB cluster must be created/configured properly.  This can be accomplished with [migrate_local_db.sh](../../build/dev/migrate_local_db.sh), as documented in the [standalone instance documentation](../../build/dev/standalone_instance.md), when using the standard standalone development DSS instance, or it can be accomplished manually with commands similar to those below starting from the repo root folder:
 
 ```bash
-go run ./cmds/db-manager \
+go run ./cmds/db-manager migration \
```

Comment:
This all seems great except I think the action is better suited to a verb ("migrate") rather than a noun ("migration") -- `db-manager migrate <options>` seems more intuitive to me than `db-manager migration <options>` since I think the latter can really only be implying one thing ("perform the 'migrate' action") whereas the latter could potentially be implying one of multiple things ("validate the ability to perform a migration", "create a new migration", "determine the status of a migration", "abort a migration").

I think the package name of `migration` is fine since all its content relates to migration, but it seems like the command should be a verb.

Reply:
Agreed